XCUITests/TopTabsTest.swift

```diff
@@ -133,6 +133,26 @@ class TopTabsTest: BaseTestCase {
         waitforExistence(app.collectionViews.cells[urlLabel])
     }
 
+    func testCloseAllTabsPrivateModeUndo() {
+        // A different tab than home is open to do the proper checks
+        navigator.toggleOn(userState.isPrivate, withAction: Action.TogglePrivateMode)
+        navigator.openURL(url)
+        waitUntilPageLoad()
+        navigator.createSeveralTabsFromTabTray (numberTabs: 3)
+        navigator.goto(TabTray)
+
+        waitforExistence(app.collectionViews.cells[urlLabel])
+        checkNumberOfTabsExpectedToBeOpen(expectedNumberOfTabsOpen: 4)
+
+        // Close all tabs, undo it and check that the number of tabs is correct
+        navigator.closeAllTabs()
+        XCTAssertTrue(app.staticTexts["Private Browsing"].exists, "Private welcome screen is not shown")
+        app.buttons["Undo"].tap()
```

Comment:
I suppose there is no accessibility Identifier for the Undo button?

Reply:
You're right, no acc.Id for that toast button